src/chrome/content/rules/FreeCallsCanada.co.uk.xml

```diff
@@ -0,0 +1,13 @@
+<ruleset name="FreeCallsCanada.co.uk">
+	<target host="freecallscanada.co.uk" />
+	<target host="www.freecallscanada.co.uk" />
+
+	<test url="http://freecallscanada.co.uk/" />
+	<test url="https://freecallscanada.co.uk/" />
+	<test url="http://www.freecallscanada.co.uk/" />
+
+	<rule from="^https?://freecallscanada\.co\.uk/" to="https://www.freecallscanada.co.uk/" />
```

Comment:
You should not touch https domains and forcing to www as long as this works server side.